tox.ini

```diff
@@ -47,9 +47,7 @@ deps =
     twtrunk: https://github.com/twisted/twisted/archive/trunk.zip
 
     {trial,coverage}: idna
-    # Pin hypothesis to 3.30 do to coverage issue
-    # See https://github.com/HypothesisWorks/hypothesis-python/issues/997
-    {trial,coverage}: hypothesis==3.30.0
+    {trial,coverage}: hypothesis>=3.44.1
```

Comment:
It'd be nice to see these dependencies moved out of `tox.ini` and into `requirements.txt` or better yet a `dev` extra.